src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+body,
+ul,
+h2,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  background-color: #cdcccc;
+}
+
+.product-card {
+  padding: 32px 0 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+
+  background-color: #fff;
+  color: #060b35;
+
+  width: 200px;
+  border-radius: 5px;
```

Comment:
you can read about best practices on formatting your styles [here](https://9elements.com/css-rule-order/)